.github/actions/build-docker/action.yml

```diff
@@ -0,0 +1,82 @@
+name: 'Docker Build'
+description: 'Builds `addons-server` docker image'
+inputs:
+  no_cache:
+    required: false
+    description: "Build fresh image without cache"
+    default: "false"
+  push:
+    required: false
+    description: "Build and push image to registry (cannot be used together with load)"
+    default: "false"
+  pull:
+    required: false
+    description: "Pull all referenced images before building"
+    default: "false"
+  load:
+    required: false
+    description: "Build and load image to local docker daemon. (cannot be used together with push)"
+    default: "false"
+  cache_dir:
+    required: false
+    description: "Docker cache directory"
+    default: /tmp/cache/docker-ci
+  python_version:
+    required: false
+    description: "Python version"
+    default: "3.10"
+
+outputs:
+  tags:
+    description: "The Docker tags for the image"
+    value: ${{ steps.meta.outputs.tags }}
+
+runs:
+  using: "composite"
+  steps:
+    # Setup docker to build for multiple architectures
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        version: latest
+        buildkitd-flags: --debug
+
+    # Login to a registry to push the image
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ github.token }}
+
+    # Determine the tags for the image
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository }}
+        tags: |
```

Comment:
We can remove the last type to make it impossible to tag latest from the action. That would improve security of this action a lot.

Reply:
Done.